lib/Arch/Context.cpp

```diff
@@ -4,33 +4,50 @@
 
 namespace remill {
 
-DecodingContext::DecodingContext(
-    std::unordered_map<std::string, uint64_t> context_value)
+
+bool DecodingContext::operator==(remill::DecodingContext const &rhs) const {
```

Comment:
Is this not the same as default? If not maybe comment why we can ignore the rest?

Reply:
You mean the default implementation of equality? they all are i think... it's just this is C++17